guides/release/getting-started/quick-start.md

```diff
@@ -77,7 +77,7 @@ In your editor, open `app/templates/application.hbs` and change it to the follow
 Ember detects the changed file and automatically reloads the page for you in the background.
 You should see that the welcome page has been replaced by "PeopleTracker".
 You also added an `{{outlet}}` to this page,
-which means that any nested route will be rendered in that place.
```

Comment:
Seems nicer to not introduce the term `nested` here when we don't use it anywhere else in the quick start